client/helpers.go

```diff
@@ -218,3 +219,28 @@ func ResolveARN(service AWSService, resourceID func(resource *schema.Resource) (
 func ResolveARNGlobal(service AWSService, resourceID func(resource *schema.Resource) ([]string, error)) schema.ColumnResolver {
 	return resolveARN(service, resourceID, false, false)
 }
+
+func stringOneOf(a string, args ...interface{}) bool {
+	for _, b := range args {
+		if b == a {
+			return true
+		}
+	}
+	return false
+}
+
+// IgnoreApiError checks if api error should be ignored
+func (c *Client) IgnoreApiError(err error) bool {
+	var ae smithy.APIError
+	if errors.As(err, &ae) {
+		errorCode := ae.ErrorCode()
+		if stringOneOf(errorCode, "ResourceNotFoundException", "WAFNonexistentItemException") ||
```

Comment:
would make this error codes a const of strings as it might grow larger overtime.